src/Persistence/ObjectManagerDecorator.php

```diff
@@ -82,6 +82,12 @@ public function initializeObject(object $obj)
         $this->wrapped->initializeObject($obj);
     }
 
+    /** @param mixed $value */
+    public function isUninitializedObject($value): bool
+    {
+        return $this->wrapped->isUninitializedObject($value);
```

Comment:
What if the wrapped object manager does not implement the method?

Reply:
Oh… right… guess I'll retarget this to 4.0.x then :disappointed:

Reply:
No, 3.4.x is fine, you just need to handle that case.

Reply:
Then to answer your original question, there will be an `Error: Call to undefined method WrappedObjectManager::isUninitializedObject()`.  Not sure how to handle it better, unless an exception is considered better than an error.

Reply:
I neither have another idea how to handle this. I'd suggest an exception too.

Reply:
Sounds like a plan.

Reply:
Ok, I'll use a custom exception with a better error message than the error then.